### PR TITLE
Auto create PR when data updated

### PR DESCRIPTION
**Pull request by fisker**
Tested on my fork.

[CI job](https://github.com/fisker/get-east-asian-width/actions/runs/7969700044/job/21755956130)
[PR](https://github.com/fisker/get-east-asian-width/pull/1)

### CODE REVIEW

File: scripts/build.js [hunk 1]
```diff
@@ -104,8 +104,8 @@ const text = await response.text();
 const parsed = parse(text);
 const functions = generateFunctions(parsed);
 
-fs.writeFileSync(
```
Comment (sindresorhus) [inline scripts/build.js:107]: Synchronous writing is significantly faster than async when you don't need concurrency, so the only way this change is beneficial is if you do the two file writings with `Promise.all`.

Comment (fisker) [reply scripts/build.js:107]: Reverted.

**Comment by sindresorhus** [comment]
Any idea why it suddenly started failing? https://github.com/sindresorhus/get-east-asian-width/actions/runs/10976824563/job/30478041494#step:6:92

**Comment by fisker** [comment]
IDK, but a manual run on my fork works  https://github.com/fisker/get-east-asian-width/actions/runs/10994838817

Try to upgrade the action?  https://github.com/peter-evans/create-pull-request/blob/main/docs/updating.md#updating-from-v6-to-v7

**Comment by fisker** [comment]
It still fails, I'll figure out if it needs permissions.

**Comment by fisker** [comment]
@sindresorhus Looks like the job still failed last week, can you check if this setting is correct? https://github.com/peter-evans/create-pull-request/issues/2767#issuecomment-1953810837

**Comment by sindresorhus** [comment]
It was not enabled. Enabled now.

**Pull request merged**